source/usage-examples/insertOne.txt

```diff
@@ -0,0 +1,22 @@
+=================
+Create a Document
+=================
+
+.. default-domain:: mongodb
+
+To create a document, define an object variable with the fields and values that you want to specify. You can insert this document using the `insertOne()
+<https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#insertOne>`_
+method. The ``insertOne()`` method inserts a document into the specified
+collection. If the specified collection does not exist, the
+``insertOne()`` method creates the collection. 
```

Comment:
Sentences 2&3 here are more or less the same. Can we combine them?

You can use `insertOne()` to add the document into a collection

Reply:
Altered sentences 2&3 to be "You can insert this document into a collection. using the insertOne() method."